src/Catty/Extension&Delegate&Protocol/Extensions/SpriteNode/CBSpriteNodePenExtension.swift

```diff
@@ -23,25 +23,49 @@
 extension CBSpriteNode {
 
     @objc func drawPenLine() {
-
-        if penConfiguration.previousPositions.last != self.position && penConfiguration.penDown {
-            penConfiguration.previousPositions.append(self.position)
+        for i in 0..<self.penConfiguration.previousPositionLines.count
```

Comment:
würde hier for element in penConfiguration.previousPositions machen anstatt einem for i loop